airflow/utils/log/file_task_handler.py

```diff
@@ -366,10 +366,7 @@ def _read(
         executor_messages: list[str] = []
         executor_logs: list[str] = []
         served_logs: list[str] = []
-        is_running = ti.try_number == try_number and ti.state in (
-            TaskInstanceState.RUNNING,
-            TaskInstanceState.DEFERRED,
```

Comment:
when the task is deferred, Does the logs are served from triggerer?

Reply:
Hi, unfortunately I don't understand the system this much to comment on this I do not know of any relations regarding the triggerer and the logs.